test/test-server/connect_handler.go

```diff
@@ -54,6 +55,86 @@ func handleConnectorResume(_ *testing.T) http.HandlerFunc {
 	}
 }
 
+func handleConnectorOffsets(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		offset := map[string]any{
+			"partition": map[string]any{
+				"server": "dbzv2",
+			},
+			"offset": map[string]any{
+				"event":          2,
+				"file":           "mysql-bin.000600",
+				"pos":            2001,
+				"row":            1,
+				"server_id":      1,
+				"transaction_id": nil,
+				"ts_sec":         1711788870,
+			},
+		}
+
+		currTime := time.Unix(1712046213, 123).UTC()
+		metadata := connectv1.NewConnectV1ConnectorOffsetsMetadata()
+		metadata.SetObservedAt(currTime)
```

Comment:
All of these variables can be directly added to the initialization below

Reply:
Done